### ISSUE TITLE
Support for fortran formatter: fprettify

### ISSUE THREAD

**Issue by argapost** [jose-elias-alvarez/null-ls.nvim#132]
### Issues

- [X] I have checked [existing issues](https://github.com/jose-elias-alvarez/null-ls.nvim/issues) and there are no existing ones with the same request.

### Feature description

Hello thank you for this plugin!

Is it possible to add the fortran auto-formatter: https://github.com/pseewald/fprettify
or should I add it manually somehow?

Thank you in advance for your help!

### Help

No

**Comment by GuillaumeAllain** [comment]
It is quite easy to add it manually, here is the solution I used to add it this morning (with my settings for capitalisation):

```.lua
require("null-ls").config({
	sources = {
		{
			method = require("null-ls").methods.FORMATTING,
			filetypes = { "fortran" },
			name = "fprettify",
			generator = require("null-ls.helpers").formatter_factory({
				command = "fprettify",
				args = { "-s", "--case", "2", "2", "2", "2" },
				format = "raw",
				to_stdin = true,
			}),
		},
	},
})
```

I don't know if I should open a PR for this @jose-elias-alvarez or is this custom solution the preferred way to add a source?

**Comment by jose-elias-alvarez** [comment]
@GuillaumeAllain Absolutely, if you think it would be useful for other people (and based on this issue it is), PRs are always welcome.
